internal/hcs/schema1/schema1.go

```diff
@@ -223,6 +223,22 @@ type GuestDefinedCapabilities struct {
 	UpdateContainerSupported      bool `json:",omitempty"`
 }
 
+func (g *GuestDefinedCapabilities) IsNamespaceAddRequestSupported() bool {
+	return g.NamespaceAddRequestSupported
+}
+
+func (g *GuestDefinedCapabilities) IsSignalProcessSupported() bool {
+	return g.SignalProcessSupported
+}
+
+func (g *GuestDefinedCapabilities) IsDumpStacksSupported() bool {
+	return g.DumpStacksSupported
+}
+
+func (g *GuestDefinedCapabilities) IsDeleteContainerStateSupported() bool {
+	return g.DeleteContainerStateSupported
+}
```

Comment:
I'm late to the party so not sure if this was discussed, but I feel we should not be putting something like this in the schema package. Ideally everything here should be code that is autogenerated, so if we change it we shouldn't add stuff that couldn't be autogenerated.

Reply:
That's reasonable. Did you have something in mind? 

Otherwise, I'll create a new package or file under hcs/ and alias the type to schema1.GuestDefinedCapabilities.

Reply:
I didn't have anything specific in mind.

Reply:
Updated. Ended up making embedded types instead and keeping them all in the gcs package.